### ISSUE TITLE
feature: Create a "core" team at the beginning, to be able to dump users without them having to accept the invitation link (OIDC Group).

### ISSUE THREAD

**Issue by Altagen** [uxbox/uxbox#3069]
### Current Workflow to enroll new users in team

When we want to collaborate in real time with Penpot, users must have been invited by email in SMTP. 
Standard flow is the following : 

- Create an account
- Create team from that account
- Go to the team
- Go to the invitation page to find other users
- Fill the email addresses and invite them 
- Users have to receive a link in email to accept the invitation and be enrolled

This is a problem for On-Premise infrastructures that do not have SMTP server and therefore cannot receive mails.

A Workaround was found by **retrieving the invitation link from the penpot console.** However, when you want to offer the service to users who do not have access to the console, an administrator must be present for each user you want to invite.

### Describe the solution you'd like.

We can imagine that all users already exist because they were created via an **IdProvider** such as **Keycloak**, **Authelia**… They are in the same group and accessible via **OIDC**. You would have to have the ability to create an **upstream team** from that group, and add all the users in that group for authentication to work in **SSO**. 

There would no need to create user accounts with passwords, only OIDC.

(related to  https://github.com/penpot/penpot/issues/1855)

### Describe alternatives you've considered.

To trigger this authentication flow it would be possible to have an API point on which to send the right instructions and which would automatically create the default team, in which users would have to be dumped, starting with an automatic acceptation, as if they had accepted the request by clicking in the email, by default.

**Comment by niwinz** [comment]
Hi @Ange-Cesari 

Penpot already comes with the ability to copy invitation links: 
![image](https://user-images.githubusercontent.com/843689/227888700-3b2694a5-2f6d-4004-82f0-3bcd2a574f38.png)

With that invitation link, the users can join using OIDC, using they accounts without setting up passwords.

**Comment by Altagen** [comment]
Hello @niwinz 

Thank you for your answer. I didn’t see the page for team settings, so I didn’t see the invitation part, which allows you to see the pending ones and copy their link.

The tip works well, I was able to create invitation links before accounts were created, and when I create those account, then authenticate, and paste the link, I am added to the team.

However, I did not find anything in the documentation to create a team in advance and be able to add users without them having to accept, using OIDC. Is that possible?


Maybe it could be possible to do it via API, or via a script (same kind as the one for adding users via the container ?) and then be able to fill the team with those users before the accounts are authenticated for the first time via SSO (OIDC) ?

**Comment by niwinz** [comment]
yep, the team creation is not implemeted, feel free to rename this issue accordingly. We we put it our backlog and will consider for future developments

**Comment by Altagen** [comment]
> yep, the team creation is not implemeted, feel free to rename this issue accordingly. We we put it our backlog and will consider for future developments

Done.
Thank you for your consideration
